app/assets/stylesheets/chat.sass

```diff
@@ -0,0 +1,17 @@
+// Place all the styles related to the Chat controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: http://sass-lang.com/
+
+
+.message-input
+  background-color: rgba(0,0,0,0.03)
```

Comment:
Take advantage of variables and create them to reference colors. For example you might create a variable here:
```
$primaryColor: rgba(0,0,0,0.03);
$redishColor: #e22424

// And use it like:

.message-input {
  background-color: $primaryColor
} 

```
Also try to use hexa colors when references in stylesheets. (prefered `#ff0000` over `rgba(255,0,0)`)

Take a look at [this guide](http://sass-lang.com/guide#topic-2) from the sass guys.